### ISSUE TITLE
Regarding doc of Linear Warmup Cosine Annealing Learning Rate Scheduler

### ISSUE THREAD

**Issue by tarun360** [PyTorchLightning/lightning-bolts#800]
In this documentation: https://pytorch-lightning-bolts.readthedocs.io/en/latest/learning_rate_schedulers.html

It's not entirely clear that whether the cosine annealing part, which was proposed in  [SGDR: Stochastic Gradient Descent with Warm Restarts](https://arxiv.org/abs/1608.03983), is with warm restarts or without it, that is, does it implement only the cosine annealing part of SGDR or does it also include the warm restart part of SGDR?
